packages/edit-site/src/components/layout/style.scss

```diff
@@ -143,10 +143,6 @@
 		top: 0;
 		bottom: 0;
 		width: 100%;
-
-		& > div {
-			border-radius: 0;
-		}
```

Comment:
The radius shouldn’t have to be zeroed here (matching a `.is-full-canvas` selector) because radius is only applied when matching `:not(.is-full-canvas)`(L136).